src/components/GlobeOptimized.jsx

```diff
@@ -602,13 +585,28 @@ export default function GlobeOptimized(props) {
     }
   }
 
+  function enableRotation(){
+    setTimeout(() => {
+      timer(function (elapsed) {
+        const { projection } = globeAttributes.current;
+        const rotation = projection.rotate();
+        const degPerMs = globeAttributes.current.rotation_speed;
+        rotation[0] -= 100 * degPerMs * 60;
+        rotation[1] -= 100 * degPerMs * 0;
+        projection.rotate(rotation);
+        globeAttributes.current.rotation = rotation;
+        render();
+      }); }, 2);
+      setTimeout(function(){
+        globeAttributes.current.rotation_speed = 0;
```

Comment:
So the rotation theoretically will happen forever but with the speed setting to `0`, it will not affect the user's interaction, is this correct?

Reply:
Yes, it is continuously rotating but after the first 5700 milliseconds the speed is 0.